public/static/locales/en/me.json

```diff
@@ -249,6 +249,9 @@
   "apiKeyMessage3": "Regenerating an API Key will invalidate your existing API Key immediately.",
   "regenerateKey": "Regenerate Key",
   "giftFund": "GiftFund",
+  "donatedTrees": "Donated Trees",
+  "remainingPaymentsText": "{{remainingPayments}} donations would take place before cancellation.",
+  "continue": "Continue"
```

Comment:
@nirali25parekh comma is missing here.

Reply:
@sagararyal The deployment failed even after fixing the error.